src/Features/Core/Portable/Completion/Providers/TypeImportCompletionItem.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Diagnostics;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.Completion.Providers
+{
+    internal static class TypeImportCompletionItem
```

Comment:
fwiw... you could just remove this static class entirely and inline this into the TypeImportCompletionProvider.

Reply:
This follows the same pattern as symbolCOmpletionItem, and I actually prefer separating the code this way :)